azure-pipelines.yml

```diff
@@ -10,7 +10,7 @@ pool:
   vmImage: 'windows-latest'
 
 variables:
-#  - group: sonarsource-build-variables
+  - group: sonarsource-build-variables
```

Comment:
to make this work, I had to check "Allow access to all pipelines" in Pipelines > Library for the variable group (and also for `maven-settings.xml`) - these are being used in templates, and there's some scope differences there, I believe